### PR TITLE
[bitnami/spring-cloud-dataflow] Remove dollar prompt from command examples in README

### PR DESCRIPTION
**Pull request by onobc**
**Description**
This removes the `$` (dollar prompt) from several copyable command examples in the `bitnami/spring-cloud-dataflow` README to allow the user to copy and used them directly without editing them to remove the `$`. This is part of an improvement effort on the Spring Cloud Dataflow Microsite to make these copyable commands easier to use. The coupling between the 2 repos is due to the fact that the microsite [includes the bitnami README](https://github.com/spring-io/dataflow.spring.io/blob/main/content/documentation/pages/1-installation/3-kubernetes/2-helm.md?plain=1#L42) via 
```
<!--TEMPLATE:https://raw.githubusercontent.com/bitnami/charts/master/bitnami/spring-cloud-dataflow/README.md-->
```

**Benefits**
Without this change, the microsite will have 4 (out of ~50) inconsistent command examples.

**Possible drawbacks**
Since this only changes the `bitnami/spring-cloud-dataflow` README, it leaves the other charts inconsistent in this area (they still have the `$ ` on the copyable command examples). 

**Applicable issues**
Supports https://github.com/spring-io/dataflow.spring.io/issues/280

>  ℹ️ ❓ I just referenced the existing dataflow.spring.io ticket via its full path. I was not sure if I should create another bitnami/charts specific issue. Please let me know if that is the case and I will amend. 

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/spring-cloud-dataflow/README.md [hunk 1]
```diff
@@ -729,7 +729,7 @@ export RABBITMQ_ERLANG_COOKIE=$(kubectl get secret --namespace default dataflow-
 Upgrade your release (maintaining the version) disabling MariaDB and scaling Data Flow replicas to 0:
 
 ```console
-$ helm upgrade dataflow bitnami/spring-cloud-dataflow --version 0.7.4 \
+helm upgrade dataflow bitnami/spring-cloud-dataflow --version 0.7.4 \
```
Comment (juan131) [inline bitnami/spring-cloud-dataflow/README.md:732]: If we remove the dollar, then we need to change the code block language:
```suggestion
```bash
helm upgrade dataflow bitnami/spring-cloud-dataflow --version 0.7.4 \
```

Comment (bono007) [reply bitnami/spring-cloud-dataflow/README.md]: Thanks for the feedback @juan131 . I have adjusted to use `bash` for the code block language. Yep, you can tell a difference when viewing it for sure. Good catch.

File: bitnami/spring-cloud-dataflow/README.md [hunk 2]
```diff
@@ -750,8 +750,8 @@ $ helm upgrade dataflow bitnami/spring-cloud-dataflow \
 
 You should see the lines below in MariaDB container logs:
 
-```console
-$ kubectl logs $(kubectl get pods -l app.kubernetes.io/instance=dataflow,app.kubernetes.io/name=mariadb,app.kubernetes.io/component=primary -o jsonpath="{.items[0].metadata.name}")
+```bash
+kubectl logs $(kubectl get pods -l app.kubernetes.io/instance=dataflow,app.kubernetes.io/name=mariadb,app.kubernetes.io/component=primary -o jsonpath="{.items[0].metadata.name}")
```
Comment (juan131) [inline bitnami/spring-cloud-dataflow/README.md:754]: I would revert the changes in this particular case. The rest of code blocks are just telling the user the command to use, however this one is showing an example including the command's output. For this particular case, using `console` + `$` is the right choice since the formatting is more appropriate.

File: bitnami/spring-cloud-dataflow/README.md (hunk 1 above)
Comment (juan131) [reply bitnami/spring-cloud-dataflow/README.md]: Thanks!

File: bitnami/spring-cloud-dataflow/README.md (hunk 2 above)
Comment (bono007) [reply bitnami/spring-cloud-dataflow/README.md]: Agreed. Adjusted.

**Review by juan131 — APPROVED** [review]
LGTM!! It seems the CI/CD it's complaining since you didn't sign your comments. This is a new mandatory requirement that we recently introduced, find more info below:

- https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work
